frontend/src/components/shared/userCard/UserCard.tsx

```diff
@@ -68,7 +62,7 @@ export const UserCard: React.FC<MainInfoProps> = ({ mainInfo }, shadowIntensity)
             );
           })}
         </div>
-        {false || loginedUser.id === mainInfo.id ?
+        {loginedUser.id === mainInfo.id ?
```

Comment:
We might end up here in situation when no user data available (as example user can clean up local storage and reload page). We need to add fallback here.